geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/DestroyRegionCommand.java

```diff
@@ -58,6 +62,12 @@ public Result destroyRegion(
       throw new EntityNotFoundException(message, ifExists);
     }
 
+    try {
```

Comment:
Why is this in gfsh command only? What happens when region is destroyed using region api? Can we destroy jdbc-mapping when region is destroyed. We do this for other functionalities, like CQs, indexes, cache writers, loaders.

Reply:
Because the only public way we have of destroying a jdbc-mapping is a gfsh command. All those other examples also have public apis that let you remove the other thing. Although at least some of your examples do not prevent a region from being destroyed (cache writer and loader; I don't know about cqs and indexes). The goal here is to help the gfsh user but not the api user.

Reply:
apis do not change cluster config. Other than that api region destroy does what it always has done. The only additional thing you would want to remove when using the apis is the async queue on the cache. Of course this is already true for the apis; if the region is the last one using the queue then destroy region does not remove that queue.